### PR TITLE
NM-192: Calculating avg commit values and trend

### PR DESCRIPTION
**Pull request by tomasmihaly**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ app.get('/teams-scores', (req, res, next) => score.getTeamsScores(req, res, next
 app.get('/user-comments-count', (req, res, next) => comments.getCommentsCountForUserName(req, res, next));
 app.get('/commits-without-request', (req, res, next) => pullRequest.getNumberOfCommitsWithoutPR(req, res, next));
 app.get('/team-collaboration', (req, res, next) => collaboration.getTeamCollaboration(req, res, next));
+app.get('/team-avg-commits', (req, res, next) => commits.getTeamAvgCommits(req, res, next));
```
Comment (SilviaKorcekova) [inline app.js:43]: It needed to be new endpoint, or it needed to be added to /team-commits? :)

Comment (jjurcik) [reply app.js:43]: Yes.
Ignore this comment, just using for testing Github Rest API, how pull request is affected by single comment.

Comment (tomasmihaly) [reply app.js]: please do review again

File: services/commits.service.js [hunk 2]
```diff
@@ -54,8 +54,9 @@ function getCommitsForTeam(teamName, dateIntervals) {
             let commitDate;
             let result = {team: teamName};
             let jiraTicketRegexp = createRegexp(utils.getJiraBoardShortcut(utils.getClientId(), teamName).toLowerCase());
-
-            return Object.assign({}, result, {numberOfCommits: getNumberOfCommitsByDateIntervals(response, dateIntervals, jiraTicketRegexp)});
+			let stats = getTeamStatsByDateIntervals(response, dateIntervals, jiraTicketRegexp, teamMembers, teamName);
+			console.log(stats);
```
Comment (SilviaKorcekova) [inline services/commits.service.js:58]: please remove console.log();

Comment (tomasmihaly) [reply services/commits.service.js]: done

**Comment by SilviaKorcekova** [comment]
Also can you please add changes to specification? :)

**Pull request merged**